third_party/move/move-model/src/metadata.rs

```diff
@@ -19,6 +19,8 @@
 const UNSTABLE_MARKER: &str = "-unstable";
 
 pub static COMPILATION_METADATA_KEY: &[u8] = "compilation_metadata".as_bytes();
+pub static DEFAULT_COMPILER_VERSION_FOR_V2: CompilerVersion = CompilerVersion::V2_0;
```

Comment:
Why do we still have these constants?

Reply:
Sorry, missed the declaration here. Removed.